.github/.wordlist.txt

```diff
@@ -48,11 +48,9 @@ algs
 alloc
 Ameba
 amebad
-AmebaD
```

Comment:
Why are these being removed?

Reply:
I ran `sort -f | uniq -i` after adding my word, The removed words were duplicates.